src/Microsoft.AspNetCore.WebHooks.Receivers.WordPress/Internal/WordPressServiceCollectionSetup.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.WebHooks.Metadata;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Microsoft.AspNetCore.WebHooks.Internal
+{
+    /// <summary>
+    /// Methods to add services for the WordPress receiver.
+    /// </summary>
+    public static class WordPressServiceCollectionSetup
```

Comment:
Suggest making this proper internal.

Reply:
Filed #233 to do this in one fell swoop.